### ISSUE TITLE
Cant change background color

### ISSUE THREAD

**Issue by humanBeer** [iantrich/text-divider-row#32]
![image](https://user-images.githubusercontent.com/47748428/208956653-d598b607-c455-42d7-89bf-6cbfe9ab5e1b.png)
Cant figure out how to change the background color to the theme background color. Been looking at it for hours. It always defaults back to the card color, but I'm using it as a standalone without a 'card', so i want it to be the default theme background color. I tried changing the .js but any changes i make don't seem to take effect no matter what i reload or restart.

Please help

**Comment by filikun** [comment]
Yeah I have the same error and card_mod does not seem to work on it either. Somewhat [duplicate of ](https://github.com/iantrich/text-divider-row/issues/21)

**Comment by ildar170975** [comment]
See no issues here (except wrong use by users).

Default dark theme:
![изображение](https://user-images.githubusercontent.com/71872483/226236460-26251ba3-a340-4fe6-b4cd-906d96f1384e.png)

Can change a background:
![изображение](https://user-images.githubusercontent.com/71872483/226236527-ce461c5a-3c2f-478c-9ccd-86d14f5b41f3.png)

As for using the card NOT inside Entities card (which is not supposed to be) - since the card does not have a `ha-card` element, you ought to use `mod-card`:
![изображение](https://user-images.githubusercontent.com/71872483/226237323-9d22a064-02ee-414c-9eb8-fe4366a78ddb.png)
